services/rds/src/main/java/org/finra/gatekeeper/services/accessrequest/AccessRequestService.java

```diff
@@ -248,7 +249,7 @@ public AccessRequestCreationResponse storeAccessRequest(AccessRequestWrapper req
         logger.info("Storing Access Request");
         accessRequestRepository.save(accessRequest);
         logger.info("Access Request stored with ID: " + accessRequest.getId());
-
+        RequestEventLogger.logEventToJson(EventType.AccessRequested, accessRequest);
```

Comment:
Move this down to the return, because it could crap out and we would be printing incorrect events